Mon Aug 10 19:15:26 EDT 2020
scc-ne6
/projectnb/peaclab-cri-cooling/EDAToolDevelopment/PACT/src /projectnb/peaclab-cri-cooling/EDAToolDevelopment/PACT/M3D_covert/scripts
lcffile = ../M3D_covert/lcf_files/M3D_lcf.csv
config file=  ../M3D_covert/config_files/M3D.config
model params file =  ../M3D_covert/modelParams_files/modelParams_M3D.config
grid file = ../M3D_covert/results/M3D/M3D_50x50.grid.steady
scp file =  ../M3D_covert/M3D_50x50.grid.steady
log file =  ../M3D_covert/logs/M3D_50x50.log
Xyce: error while loading shared libraries: libloca.so.12: cannot open shared object file: No such file or directory
Xyce: error while loading shared libraries: libloca.so.12: cannot open shared object file: No such file or directory
Xyce: error while loading shared libraries: libloca.so.12: cannot open shared object file: No such file or directory
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
Xyce: error while loading shared libraries: libloca.so.12: cannot open shared object file: No such file or directory
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[29048,1],0]
  Exit code:    127
--------------------------------------------------------------------------
chipstack length 0.001
chipstack width 0.001
64 x 64 ; grid_length and width: 1.5625e-05 1.5625e-05
318.15
high-level solver = SPICE_transient
low-level solver = TRAP
Traceback (most recent call last):
  File "PACT.py", line 627, in <module>
    grid_temperature = solver.getTemperature(solver_properties)
  File "/projectnb/peaclab-cri-cooling/EDAToolDevelopment/PACT/src/SPICESolver_transient.py", line 386, in getTemperature
    reshape_x = tmp.reshape(self.nl,self.nr,self.nc)
ValueError: cannot reshape array of size 3328 into shape (13,64,64)
/projectnb/peaclab-cri-cooling/EDAToolDevelopment/PACT/M3D_covert/scripts
Mon Aug 10 19:15:36 EDT 2020
['50x50']
4
